iree/base/api.c

```diff
@@ -696,6 +695,19 @@ iree_status_allocate_vf(iree_status_code_t code, const char* file,
 #endif  // has IREE_STATUS_FEATURE_ANNOTATIONS
 }
 
+IREE_API_EXPORT IREE_MUST_USE_RESULT iree_status_t IREE_API_CALL
+iree_status_allocate_vf(iree_status_code_t code, const char* file,
+                        uint32_t line, const char* format, va_list varargs) {
+  // make a copy so we can walk the arguments once to determine storage size
+  // and once to copy the values.
```

Comment:
```suggestion
  // Make a copy so we can walk the arguments once to determine storage size
  // and once to copy the values.
```

style nit: capitalize the first word in the comment